multibindings.md

```diff
@@ -22,9 +22,10 @@ In order to contribute one element to an injectable multibound set, add an
 ```java
 @Module
 class MyModuleA {
+String d = "D";
   @Provides @IntoSet
   static String provideOneString(DepA depA, DepB depB) {
-    return "ABC";
+    return depA.getA() + new Obj(depB).getB() + "C" + depA.getD(d);
```

Comment:
I like that this actually uses the deps now, but maybe just keep it a little more straightforward and do `depA.toString() + depB.toString()`